sahi/model.py

```diff
@@ -1063,3 +1062,102 @@ def _create_object_prediction_list_from_original_predictions(
             object_prediction_list_per_image.append(object_prediction_list)
 
         self._object_prediction_list_per_image = object_prediction_list_per_image
+
+
+@check_requirements(["tensorflow", "tensorflow_hub"])
+class TensorflowHubDetectionModel(DetectionModel):
+    def load_model(self):
+        import tensorflow_hub as hub
+
+        if self.model_path[: len("https://tfhub.dev/tensorflow")] == "https://tfhub.dev/tensorflow":
```

Comment:
soyle daha uygun olur:
```suggestion
        if 'tfhub.dev/tensorflow' not in self.model_path:
```